.changeset/easy-vans-laugh.md

```diff
@@ -0,0 +1,7 @@
+---
+'astro': minor
```

Comment:
How confident are we there will be no integrations impacted in a breaking way so this is safe in a minor release? Would we revert if people report this broke their projects?

Reply:
Try to look at it the other way around: if there are integrations that create invalid configurations it will result in a buggy functionality, and that bug isn't caught by Astro now. The integration needs to fix it.

This means that there's a bug nonetheless. With this, the bug is caught by the users every time, and reported to integration developers.

Reply:
And now there's a log showing what integration is failing to match the validation so it's easier for users to report

Reply:
Yeah, more thinking of the scenario where being able to set some technically “invalid” config ended up being useful for someone. But as long as we’re confident this shouldn’t happen/are prepared to handle reports of it happening that sounds fine to me.